### DISCUSSION TITLE
QDrawer in component

### DISCUSSION THREAD

**Discussion by dajermstad** [quasarframework/quasar#11926] [General - Components / Directives / etc] [answered]
Hi.
Is it possible somehow to have a Qdrawer in custom components?
 For instance if I want to have a section with a drawer in it. 
 
 Im only able to create it for the whole page. 

Or is it some other way to create a drawer inside a custom component?

best regards

**Comment by pdanpdan** [comment, marked as answer]
Containerized QLayout

**Reply by dajermstad** [reply]
Thanx, worked out well

**Comment by seiyria** [comment]
Is there an example of this anywhere @pdanpdan?

**Comment by pdanpdan** [comment]
in the docs (https://quasar.dev/layout/layout#containerized-qlayout)
